Kerberos.NET/Crypto/DecryptedKrbApRep.cs

```diff
@@ -62,13 +62,13 @@ public override void Validate(ValidationActions validation)
                 );
             }
 
-            if (this.SequenceNumber != this.Response.SequenceNumber)
-            {
-                throw new KerberosValidationException(
-                    $"SequenceNumber does not match. Sent: {this.SequenceNumber}; Received: {this.Response.SequenceNumber}",
-                    nameof(this.SequenceNumber)
-                );
-            }
+            //if (this.SequenceNumber != this.Response.SequenceNumber)
```

Comment:
@SteveSyfuhs Would you like us to remove `DecryptedKrbApRep_Validate_Sequence` or try to make it work by passing in a custom validator with this implementation as the default?